scripts/populate_annex_scientificname.py

```diff
@@ -11,25 +13,32 @@ def _get_annex(path):
     """
     with open(path) as csvfile:
         annex_data = reader(csvfile)
-        scientificnames_annex = dict()
+        annex_scientificnames = dict()
         fields = next(annex_data)
         print("Columns in " + path + ": " + ", ".join(fields))
         for (i, row) in enumerate(annex_data):
             id = i + 1
             scientific_name_original = row[1]
             scientific_name_corrected = row[2]
+            authorship = row[3]
             annex_id = row[0]
-            remarks = row[4]
-            scientificnames_annex[id] = {'id': id,
-                                         'scientificNameOriginal': scientific_name_original,
+            remarks = row[5]
+            is_scientific_name = True
+            if scientific_name_corrected == '':
+                is_scientific_name = False
+            annex_scientificnames[id] = {'id': id,
```

Comment:
@damianooldoni: I notice you set the ID here in Python code, and it this ID is later pushed to the database like other fields... I think it would be a bit cleaner to not give any ID to the insert statement, so we let Postgres choose it (the field is set as a serial for this)

Reply:
You are right. Do you change it or I?

Reply:
I can do that!

Reply:
Done, note that I also renamed `get_annex()` and simplified its implementation. 

In particular, it was returning a dict (key: ID, value: dict_of_other fields). It appeared the keys were actually not used later, only the values. So I simplified this data structure as a List of dicts.